wiki/Options/en.md

```diff
@@ -2,171 +2,165 @@
 
 ## General
 
 ![General icon](img/general.jpg "General icon")
 
-This section is about your account, language settings, and what updates you want your osu!client to receive.
+This section is about your account, language settings, and what updates you want your game client to receive.
 
 ### Sign In
 
-If you are not signed into the osu!client, the game will automatically open the options menu, prompting you to sign in.
-You can ignore this by pressing `Esc`, clicking on the back button, or navigating to the song selection screen.
-By not signing in, the osu!client will mark you as a "Guest" and you will not be able to fetch online scores or play [Multi](wiki/Multi).
+If you are not signed into the game client, it will automatically open the options menu, prompting you to sign in. You can ignore this by pressing `Esc`, clicking on the back button, or navigating to the song selection screen. By not signing in, the game client will mark you as a "Guest" and you will not be able to fetch online scores, submit scores, or play [Multi](wiki/Multi).
 
-If you are a guest, you will see these options:
+If you are a guest, these options will be shown:
 
-| Name                | Effect                                                                                           | Type     | Default   |
-|---------------------|--------------------------------------------------------------------------------------------------|----------|-----------|
-| `Username`          | Type in your registered _osu!_ username.                                                         | Text     | _(Empty)_ |
-| `Password`          | Type in your password associated with your _osu!_ username.                                      | Text     | _(Empty)_ |
-| `Remember Username` | If checked, this osu!client will remember the username.                                          | Checkbox | `Enabled` |
-| `Remember Password` | If checked, Have the client remember the password.                                               | Checkbox | `Enabled` |
-| `Sign In`           | Start signing yourself in. You can also press `Enter` in the username or password forms.         | Button   |           |
-| `Create an account` | Redirects you to [Create Account](https://osu.ppy.sh/p/register) using your default web browser. | Button   |           |
+| Name | Effect | Type | Default |
+| --- | --- | --- | --- |
+| `Username` | Type in your registered osu! username. | Text | *(empty)* |
+| `Password` | Type in your password associated with your osu! username. | Text | *(empty)* |
+| `Remember Username` | If checked, this client will remember the username. | Checkbox | `Enabled` |
+| `Remember Password` | If checked, Have the client remember the password. | Checkbox | `Disabled` |
+| `Sign In` | Start signing yourself in. Pressing `Enter` in the username or password forms also works. | Button |  |
```

Comment:
Can u pluralize the verbs (i.e. `Start` -> `Starts`) every instance it is singular? It changes it to an effect rather than a command. It also sounds better.